docs/guides/create-composable-components.stories.mdx

```diff
@@ -0,0 +1,187 @@
+import { Meta } from "@storybook/addon-docs";
+
+<Meta title="Guides/Creat Composable Components" />
+
+# Create Composable Components
+
+### Why Composability Matters
+
+**Feedback from Atlantis consumers has highlighted that**:
+
+- Developers want to make minor modifications without requesting feature
+  additions.
+- Designers need more flexibility to tailor components to specific use cases.
+- Current rigidity leads to workarounds, custom builds, and decreased adoption
+  of Atlantis.
```

Comment:
given Atlantis is open to the world kinda feels like we're airing some of our dirty laundry 😅